Gruntfile.js

```diff
@@ -16,6 +16,8 @@ module.exports = (grunt) => {
 
     grunt.initConfig(tasks);
     grunt.registerTask('build', ['cssmin', 'uglify']);
+    grunt.registerTask('watch', ['watch']);
+    grunt.loadNpmTasks('grunt-contrib-watch');
```

Comment:
But without it here command 'grunt watch' doesn't work at me correctly. 
And official documentation (https://github.com/gruntjs/grunt-contrib-watch)  says 

npm install grunt-contrib-watch --save-dev
Once the plugin has been installed, it may be enabled inside your Gruntfile with this line of JavaScript:
grunt.loadNpmTasks('grunt-contrib-watch');

I'll better remove it from file /grunt/watch.task.js

Reply:
I meant this line: `grunt.loadNpmTasks('grunt-contrib-watch');` which is already loaded in the drop-in task.